### ISSUE TITLE
The ngbDropdownToggle doesn't work when its content is wrapped in another element

### ISSUE THREAD

**Issue by Tragetaschen** [ng-bootstrap/ng-bootstrap#836]
Please remember, the issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
Please read https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.

### Bug description:

This was very likely introduced with #785. The click event doesn't originate from the `ngbDropdownToggle`, but from one of its children. A click event first opens the dropdown and then (while bubbling out) immediately closes it again.

### Link to minimally-working plunker that reproduces the issue:

You can use this template as a starting point: http://plnkr.co/edit/i3ajJvZ3bjQyk2I3rDmy?p=preview

In the dropdown documentation example, replace
```
<button class="btn btn-outline-primary" id="dropdownMenu1" ngbDropdownToggle>Toggle dropdown</button>
```
with
```
<button class="btn btn-outline-primary" id="dropdownMenu1" ngbDropdownToggle><strong>Toggle dropdown</strong></button>
```

### Version of Angular, ng-bootstrap, and Bootstrap:

Angular: 2.0.0

ng-bootstrap: 1.0.0-alpha.6

Bootstrap: -

**Comment by pkozlowski-opensource** [comment]
I believe that this was fixed in master already via fdf8d4b164135e5b5891f3a37620891f7a7ebe21 - will be part of the next release (this week). Closing as a duplicate of #803

**Comment by Tragetaschen** [comment]
Yup, looks like it :)
